CHANGELOG.rst

```diff
@@ -1,7 +1,51 @@
 Version 0.17.0 ()
 ---------------------------
 
-NOTE: This drops pillow as a required dependency.
+Breaking Changes:
+
+- Drop support for Python 3.9.
+- ``pillow`` and ``dask`` are no longer required dependencies.
+- Moved to ``pixi`` and ``hatchling``.
+
+Enhancements:
+
+- Edge bundling speedup using optimized ``numba`` code by `@lmcinnes <https://github.com/lmcinnes>`_ (`#1383 <https://github.com/holoviz/datashader/pull/1383>`_).
+- Drop Python 3.9 by `@hoxbro <https://github.com/hoxbro>`_ (`#1399 <https://github.com/holoviz/datashader/pull/1399>`_).
```

Comment:
Let us remove this as it wasn't an enhancement. I wrongly classified it as an enhancement.